packages/core/src/js/replay/mobilereplay.ts

```
@@ -0,0 +1,170 @@
+  if (notMobileOs()) {
+    logger.warn(`[Sentry] ${MOBILE_REPLAY_INTEGRATION_NAME} is not supported on this platform.`);
+  }
+
+  if (isExpoGo() || notMobileOs()) {
+    return mobileReplayIntegrationNoop();
+  }
+
+  const options = { ...defaultOptions, ...initOptions };
+
+  async function processEvent(event: Event): Promise<Event> {
+    const hasException = event.exception && event.exception.values && event.exception.values.length > 0;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)